Gems/AWSCore/Code/Platform/iOS/platform_ios_files.cmake

```diff
@@ -10,4 +10,5 @@ set(FILES
     AWSCore_Traits_Platform.h
     AWSCore_Traits_iOS.h
     ../Common/GetCertsPath_Null.cpp
+    ../Common/GetJsonSchemaPath_Common.cpp
```

Comment:
Is the file found on iOS?

Reply:
Unfortunately we haven't gotten a chance to test this on iOS yet. We can do the exact same thing to read the schema from assets as Android if that's the case.